src/openzeppelin/account/account.cairo

```diff
@@ -5,93 +5,91 @@ use serde::Serde;
 use starknet::ContractAddress;
 
 use openzeppelin::account::interface::Call;
-use openzeppelin::utils::serde::SpanSerde;
 
 const TRANSACTION_VERSION: felt252 = 1;
+
 // 2**128 + TRANSACTION_VERSION
 const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457;
 
-#[abi]
-trait AccountABI {
-    #[external]
-    fn __execute__(calls: Array<Call>) -> Array<Span<felt252>>;
-    #[external]
-    fn __validate__(calls: Array<Call>) -> felt252;
-    #[external]
-    fn __validate_declare__(class_hash: felt252) -> felt252;
-    #[external]
+#[starknet::interface]
+trait AccountTrait<TState> {
+    fn __execute__(self: @TState, calls: Array<Call>) -> Array<Span<felt252>>;
+    fn __validate__(self: @TState, calls: Array<Call>) -> felt252;
```

Comment:
I was trying to use Trait instead of ABI as we had before, but maybe ABI is still a better name. I don't want to use the interface naming convention for this ABI intended traits, because is confusing IMO. What is the ERC721 interface, the one from the EIP, or the one from our implementation? I like using ABI for the implementation and interface for the EIP one. Wdyt?

Reply:
> What is the ERC721 interface, the one from the EIP, or the one from our implementation?

Haha right, I got you

>  I like using ABI for the implementation and interface for the EIP one. Wdyt?

Yeah, I think that's a better approach. Otherwise, I think this only stirs the trait vs interface confusion